api/capi/include/nnstreamer.h

```diff
@@ -128,6 +128,7 @@ typedef enum {
   ML_NNFW_TYPE_OPENVINO = 6,          /**< Intel openVINO. */
   ML_NNFW_TYPE_VIVANTE = 7,           /**< VeriSilicon's Vivante (TBD) */
   ML_NNFW_TYPE_EDGE_TPU = 8,          /**< Google Coral edge TPU (USB) */
+  ML_NNFW_TYPE_CAFFE = 9,             /**< Caffe (.caffemodel) */
```

Comment:
caffe? armnn? which one is correct?
IMO, ML_NNFW_TYPE_ARMNN with armnn-subplugins

Reply:
Updated to armnn.